### DISCUSSION TITLE
nestjs-unit-testing

### DISCUSSION THREAD

**Discussion by giscus** [tomwray13/tomray.dev#62] [Blog Comments]
# nestjs-unit-testing

A deep dive tutorial into how to apply unit tests in NestJS with Jest, including how to mock with test doubles and other advanced testing strategies.

https://www.tomray.dev/nestjs-unit-testing

**Comment by Maetes** [comment]
Really appreciate this and especially all the other posts about nest. They are one of the best, cover alot and are easy to follow <3

**Reply by tomwray13** [reply]
Thanks so much @Maetes! A lot of work goes into these blog posts so this kind of feedback makes it all worth it 😌

**Comment by MateusPedrosoSilva** [comment]
Thanks man, your content is great!!!

**Comment by justin-rignault** [comment]
Good stuff! Very clear and detailed
